src/index.ts

```diff
@@ -125,3 +125,4 @@ export { default as dialogBehavior } from './lib/accessibility/Behaviors/Dialog/
 // Utilities
 //
 export { default as mergeThemes } from './lib/mergeThemes'
+export { createComponent } from './lib'
```

Comment:
I'm not sure if we should we export some types as well?

Reply:
Not sure on this either, maybe we should add them..

Reply:
I exported them

Reply:
Always export all typings that are for the public API.